### ISSUE TITLE
Both 5V and 3V are hooked up?

### ISSUE THREAD

**Issue by RobThree** [spuder/OpenSpool#6]
I'm confused. Why is **both** 3V and 5V hooked up to the ESP32-S3 in [your diagram](https://github.com/spuder/OpenSpool/blob/26bc6550290323e55f7e9be442d46adbeaa9df4e/images/OpenSpoolMiniWiringDiagram2.png?raw=true)?

![image](https://github.com/user-attachments/assets/b0ee2743-431c-4434-8278-4874d5345b36)

Isn't it enough to hook up just 5V? It would save a levelshifter (or even a 4-levelshifter module that could be a single levelshifter module).

**Comment by spuder** [comment]
The level shifter is only for the data line of the led. Since the LED needs 5v logic but the esp32 GPIOs only provide 3.3v some LEDS are unreliable with the lower voltage. 


On the PCB version, the 4 line shifter has been replaced with a single transistor and 2 resistors. 
<img width="375" alt="Screenshot 2024-12-07 at 09 18 41" src="https://github.com/user-attachments/assets/3358194b-288e-4393-896d-ce8f915571dd">

**Comment by RobThree** [comment]
> The level shifter is only for the data line of the led. Since the LED needs 5v logic but the esp32 GPIOs only provide 3.3v some LEDS are unreliable with the lower voltage.

I wasn't asking what the levelshifter is for. In the diagram you're hooking up **both** 5V (bottom right) and 3V (bottom left) to the Wemos D1, See the arrows. That doesn't make sense. And _that_ makes the "second" levelshifter redundant.

**Comment by RobThree** [comment]
Wait, wait, wait... I'm a dumbass....

You're correct. You need to get the 3V from somewhere. It's coming FROM the Wemos TO (feed) the levelshifter.... 

Jeez. My bad.

**Comment by spuder** [comment]
The quad level shifter is overkill. 
The single level shifters I found on amazon were actually more expensive than the quads so I linked the cheaper versions. 

Here is the schematic I'm using. It seems to work, but if it can be improved I'll gladly change it. 

<img width="256" alt="Screenshot 2024-12-07 at 14 22 19" src="https://github.com/user-attachments/assets/abafd19c-b6e7-4f88-bbd7-ea7a35806b52">
